src/sync/impl/sync_metadata.hpp

```diff
@@ -214,7 +214,8 @@ friend class SyncFileActionMetadata;
     /// exception will be thrown.
     SyncMetadataManager(std::string path,
                         bool should_encrypt,
-                        util::Optional<std::vector<char>> encryption_key=none);
+                        util::Optional<std::vector<char>> encryption_key=none,
+                        bool should_recheck_keychain=false);
```

Comment:
`SyncMetadataManager` is internal to object store, and it doesn't look like any callers were updated to pass this argument. How is this code path ever activated?